resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
+                "exp": 10,
+                "stat_trait": ["cold","strict","grumpy","lonesome","gloomy"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l lashes {PRONOUN/p_l/poss} tail. Of course app1 will never be a warrior if {PRONOUN/app1/subject} {VERB/app1/think/thinks} like <i>that.</i> c_n doesn't want warriors who think like <i>that.</i> Half of glory, {PRONOUN/p_l/subject} {VERB/p_l/explain/explains} to app1, is believing you deserve it. P_l challenges app1: focus on <i>that</i>. When p_l and app1 return to the training grounds, the other trainees notice app1 fighting with a newfound vigor.",
+                "exp": 10,
+                "stat_trait": ["fierce","bold","confident","competitive","ambitious","rebellious","daring","adventurous","cunning"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l's heart lurches seeing the apprentice so low. Could p_l have done something to make app1 feel this way? Not having any words p_l sits close to app1, wrapping {PRONOUN/p_l/poss} tail around {PRONOUN/app1/poss}. P_l purrs softly, reassuringly. Until app1 purrs back, gratefully. ",
+                "exp": 10,
+                "stat_trait": ["insecure","nervous","oblivious","careful","responsible"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l shifts awkwardly at app1's side until {PRONOUN/p_l/subject} can't stand the silence any longer. P_l drops into a crouch and nudges app1 with {PRONOUN/p_l/poss} paw. App1 looks back, confused as p_l keeps bouncing and poking at {PRONOUN/p_l/poss} with a goofy expression on {PRONOUN/p_l/poss} face, until app1 gives in. The warrior and apprentice play like kits, and the smile on app1's face says {PRONOUN/app1/subject} {VERB/app1/have/has} forgotten all about their doubt.",
```

Comment:
SO CUTE

might need to be a little shorter tho

also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l

can't capitalise app1 or p_l